.gitattributes

```diff
@@ -0,0 +1 @@
+.gitignore export-ignore
```

Comment:
Should also ignore the .gitattributes file too.

Reply:
You can but you don’t need to. The process seems to self exclude itself

Reply:
No it doesn't. You need to exclude the file, otherwise it will show up. I just downloaded the zip from your branch to confirm:

<img width="819" alt="image" src="https://user-images.githubusercontent.com/2829600/68619437-6651eb80-04c3-11ea-8f59-c3a831a23c0e.png">

Reply:
I'm going to blame jetlag/being tired... That and most places have added it already anyway by default (so I'm probably just glazing over when reading). But yeah, you're right